README.md

```diff
@@ -176,6 +192,28 @@ Your should see report file generated as ExtentReport.html under the target fold
 ##WIP
 iOS WebTest using Safari
 
+##FAQ
+Q. Is this framework supports to run multiple IOS simulators?
+
+A. No, Apple does not allow to do so.
```

Comment:
@SrinivasanTarget yup they do, so ideally appium would also get this in up coming releases with help of WebDriverAgent